changelog/_unreleased/2024-11-28-new-adresse-selection-modal.md

```diff
@@ -0,0 +1,35 @@
+---
+title: New adresse manager modal
+issue: NEXT-19776
+---
+# Core
+* Added `FEATURE_NEXT_19776` feature flag to enable/disable the new address manager modal and account address page redesign 
```

Comment:
Is this the standard for feature flags? To use the ticket number? If not, then I'd rather make it something like `FEATURE_IMPROVED_ADDRESS_MODAL`